crates/cxx-qt-lib-extras/Cargo.toml

```diff
@@ -11,7 +11,7 @@ license.workspace = true
 description = "Extra Qt types for integrating `cxx-qt` crate with `cxx` that are not available in `cxx-qt-lib`"
 repository.workspace = true
```

Comment:
the build fails because i believe you need to update the reexport links. Search for 

`.reexport_dependency` and update those names in the `Interface` usages